src/middleware.ts

```diff
@@ -131,6 +140,46 @@ export function middleware(req: NextRequest, ev: NextFetchEvent) {
 		})
 	}
 
+	/**
```

Comment:
@BRKalow looks like this could be fixed by moving the happykit logic to https://github.com/hashicorp/dev-portal/pull/1190/files#diff-0626369278c89781505d2c04865ecce1302e979271bcfb471e029c19addc13a0R117

Reply:
Works for me!

Reply:
https://github.com/hashicorp/dev-portal/pull/1190/commits/24dd606b333c3ec11e03b5d9f0ab94772984084d